core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -603,10 +613,18 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
     }
   }
 
+  private def attributeIsEmpty(attribute: Attribute): Boolean =
+    attribute match {
+      case AttributeNull          => true
+      case AttributeString(value) => value == ""
+      case _                      => false
+    }
```

Comment:
to be Scala-idiomatic, include the "if" clause in the `case` definition:
```scala
  private def attributeIsEmpty(attribute: Attribute): Boolean =
    attribute match {
      case AttributeNull                         => true
      case AttributeString(value) if value == "" => true
      case _                                     => false
    }
```
I'm personally fine with either approach, just bringing it up for discussion/your choice